vendor/knative.dev/pkg/injection/sharedmain/main.go

```diff
@@ -436,3 +443,41 @@ func ControllersAndWebhooksFromCtors(ctx context.Context,
 
 	return controllers, webhooks
 }
+
+// CheckMinimumKubeVersion checks if current K8s version we are on is higher than the one passed.
+// If an error is returned then the version is not higher than the minimum
+func CheckMinimumKubeVersion(versioner discovery.ServerVersionInterface, version string) error {
```

Comment:
why adding the func on this file?

the `pkg/version` has already the `normalizeVersion` func

Reply:
Does not do what I want so I adapted it, we also have the same at the S-O side.